lisa/spine.sh

```diff
@@ -0,0 +1,200 @@
+#!/bin/bash
+# Written by Lisa Eunyoung Lee - Jun 2022
+# Revised by Lisa Eunyoung Lee - Jan 2023
+# Spinal Cord Toolbox - Version 5.7
+
+# T2 - Spinal Cord CSA averaged between C2-C4
+# MT - Spinal Cord MTR averaged between C2-C4
+
+# Check input arguments
+
+if [ $# -eq 0 ] ; then
+    echo "spine.sh"
+	echo "sh spine.sh SITE GROUP NUMSUBS MONTH"
+	printf "Options: \n SITE = 01 (SMH Toronto), 02 (UBC Vancouver), 03 (CHUM Montreal), 04 (UCAL Calgary), 05 (UALB Edmonton) \n GROUP = RIS (RIS), RRM (RRMS), PPM (PPMS), CON (control) \n NUMSUBS = subject number (e.g. '2 56 105') \n MONTH = 0 (baseline), 12 (month 12), 24 (month 24), 48 (month 48) \n"
+	exit 1
+elif [ $# -ne 4 ] ; then
+	echo "spine.sh: Incorrect number of inputs, $# provided but 4 required." 1>&2
+	exit 1
+fi; 
+
+# Set up variables
+
+SITE=$1
+if [ $SITE = '01' ]; then STUDYSITE='Toronto'
+elif [ $SITE = '02' ]; then STUDYSITE='Vancouver'
+elif [ $SITE = '03' ]; then STUDYSITE='Montreal'
+elif [ $SITE = '04' ]; then STUDYSITE='Calgary'
+elif [ $SITE = '05' ]; then STUDYSITE='Edmonton'
+else echo "Site type not recognized. Only '01', '02', '03', '04' or '05' are accepted." 1>&2;exit 1
+fi
+GROUP=$2
+NUMSUBS=$3
+MONTH=$4
+
+for NUMSUB in $NUMSUBS;do
+	if [ $SITE = '01' ] || [ $SITE = '02' ] || [ $SITE = '03' ] || [ $SITE = '04' ] || [ $SITE = '05' ]; then NUM=$(printf "%03d\n" $NUMSUB)
+	fi
+
+# Set up paths
+
+	data="/Volumes/Neuron/Lisa/canproco/data/spine/$STUDYSITE"
+	ubc="/Volumes/Neuron/Lisa/canproco/data/ubc/$STUDYSITE/CAN-$SITE-$GROUP-$NUM-M${MONTH}/cache"
+	qc="/Volumes/Neuron/Lisa/canproco/data/spine/qc_SCT/M${MONTH}/$STUDYSITE"
```

Comment:
Fixed paths shouldn't be in the code. Those paths should be given in the command line, the same way you give `SITE`, `GROUP` ...